crates/uv/src/target.rs

```diff
@@ -0,0 +1,168 @@
+            Self::X8664AppleDarwin => Platform::new(
+                Os::Macos {
+                    major: 10,
+                    minor: 12,
+                },
+                Arch::X86_64,
+            ),
+            Self::Aarch64AppleDarwin => Platform::new(
+                Os::Macos {
+                    major: 11,
+                    minor: 0,
+                },
+                Arch::Aarch64,
+            ),
+            Self::Aarch64UnknownLinuxGnu => Platform::new(
+                Os::Manylinux {
+                    major: 2,
+                    minor: 17,
+                },
+                Arch::Aarch64,
+            ),
+            Self::Aarch64UnknownLinuxMusl => {
+                Platform::new(Os::Musllinux { major: 1, minor: 2 }, Arch::Aarch64)
+            }
+            Self::X8664UnknownLinuxMusl => {
+                Platform::new(Os::Musllinux { major: 1, minor: 2 }, Arch::X86_64)
+            }
+        }
+    }
+
+    /// Return the `platform_machine` value for the target.
+    pub(crate) fn platform_machine(self) -> &'static str {
+        match self {
+            Self::X8664PcWindowsMsvc => "x86_64",
+            Self::X8664UnknownLinuxGnu => "x86_64",
+            Self::X8664AppleDarwin => "x86_64",
+            Self::Aarch64AppleDarwin => "aarch64",
+            Self::Aarch64UnknownLinuxGnu => "aarch64",
+            Self::Aarch64UnknownLinuxMusl => "aarch64",
+            Self::X8664UnknownLinuxMusl => "x86_64",
+        }
+    }
+
+    /// Return the `platform_system` value for the target.
+    pub(crate) fn platform_system(self) -> &'static str {
+        match self {
+            Self::X8664PcWindowsMsvc => "Windows",
+            Self::X8664UnknownLinuxGnu => "Linux",
+            Self::X8664AppleDarwin => "Darwin",
+            Self::Aarch64AppleDarwin => "Darwin",
+            Self::Aarch64UnknownLinuxGnu => "Linux",
+            Self::Aarch64UnknownLinuxMusl => "Linux",
+            Self::X8664UnknownLinuxMusl => "Linux",
+        }
+    }
+
+    /// Return the `platform_version` value for the target.
+    pub(crate) fn platform_version(self) -> &'static str {
+        match self {
+            Self::X8664PcWindowsMsvc => "",
+            Self::X8664UnknownLinuxGnu => "",
+            Self::X8664AppleDarwin => "",
+            Self::Aarch64AppleDarwin => "",
+            Self::Aarch64UnknownLinuxGnu => "",
+            Self::Aarch64UnknownLinuxMusl => "",
+            Self::X8664UnknownLinuxMusl => "",
+        }
+    }
+
+    /// Return the `platform_release` value for the target.
+    pub(crate) fn platform_release(self) -> &'static str {
+        match self {
+            Self::X8664PcWindowsMsvc => "",
+            Self::X8664UnknownLinuxGnu => "",
+            Self::X8664AppleDarwin => "",
+            Self::Aarch64AppleDarwin => "",
+            Self::Aarch64UnknownLinuxGnu => "",
+            Self::Aarch64UnknownLinuxMusl => "",
+            Self::X8664UnknownLinuxMusl => "",
```

Comment:
Per the spec, you're allowed to leave these empty to indicate "unknown".

It looks like this is sometimes used though: https://github.com/search?type=code&q=%22platform_release%22+path%3Arequirements.txt. E.g., sometimes as: `sys_platform == "win32" and platform_release == "10"`.